.dev/compare_branches.R

```diff
@@ -49,24 +51,35 @@ if (!interactive()) {
 param_list <- list(
   optparse::make_option(
     "--linters",
-    default = "object_usage_linter",
-    help = "Run the comparison for these linter(s) (comma-separated) [default %default]"
+    default = if (interactive()) {
+      readline("Provide a comma-separated list of linters to compare: ")
+    },
+    help = "Run the comparison for these linter(s) (comma-separated)"
   ),
   optparse::make_option(
     "--branch",
     default = if (interactive()) {
-      readline("Name a branch to compare to master (or skip to enter a PR#): ")
+      readline("Name a branch to compare to the base branch (or skip to enter a PR#): ")
```

Comment:
maybe default to assignment_linter in batch mode?

Reply:
or maybe just error if not provided in batch mode? is that what happens when default=NULL?

Reply:
Crude error messages I think.
`stop()` is probably the best thing to do.